MahApps.Metro/Controls/MetroWindow.cs

```diff
@@ -184,33 +181,6 @@ public WindowCommandsOverlayBehavior IconOverlayBehavior
         }
 
         /// <summary>
-        /// Gets/sets the style for the MIN button style.
-        /// </summary>
-        public Style WindowMinButtonStyle
-        {
-            get { return (Style)this.GetValue(WindowMinButtonStyleProperty); }
```

Comment:
Not usable for 1.2.0 or not usable **until** 1.2.0. If you want I can add them back as an override but set them to null by default since Light and Dark styles are now overridable. But this would be rather dumb since the whole point of Light and Dark styles are for them to be overridable. I could possibly make it override both Light and Dark styles of WindowButtonCommands but again seems dumb. You seem like you don't want any changes to occur. While I agree to a point, if something is bad or unnecessary I don't see why can't we delete it.